open-policy-agent/authorization/disallow-scc-runasany/template.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8snorunasanyscc
+  annotations:
+    description: Requires custom SCCs to not have the `RunAsAny` type for the `runAsUser` attribute.
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sNoRunAsAnySCC
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8snorunasanyscc
+
+        violation[{"msg": msg}] {
+          input.review.object.kind == "SecurityContextConstraints"
+          allowedSCCs := { scc | scc := input.parameters.allowedSCCs[_]}
+          sccNameSet := { input.review.object.metadata.name }
+          validationSet := allowedSCCs - sccNameSet
+          count(validationSet) == count(allowedSCCs)
```

Comment:
i'm pretty sure that there is a clearer way to check if sccNameSet is not in sccNameSet, it's really confusing now

Reply:
@cloudlabmaz Agreed, will work on a simplified solution